packages/jest-circus/src/__tests__/__snapshots__/after_all.test.js.snap

```diff
@@ -13,16 +13,25 @@ finish_describe_definition: describe
 run_start
 run_describe_start: ROOT_DESCRIBE_BLOCK
 run_describe_start: describe
+hook_start: beforeAll
+> beforeAll
```

Comment:
we even had this snapshot test that captured that hooks will be registered but won't run 😄